### PR TITLE
Fix: UTType init returns nil on M1 Mac

### PR DESCRIPTION
**Pull request by billypchan**
### Issues

When running test on M1 Mac iOS simulator, `UTType(mimeType: )` always return nil and cause `hasRasterImage` always false

### Causes

Unknown

### Solutions

As a workaround, check `mimeType` is SVG to return it is a raster image or not.

related PR:
https://github.com/wireapp/wire-ios-images/pull/43

### CODE REVIEW

**Review by David-Henner — APPROVED** [review]

**Comment by billypchan** [comment]
I checked that this issue is related to a bug of arm64 simulator. But I would keep this temporary fix til arm64 simulator frameworks are ready.

**Comment by typfel** [comment]
@billypchan I don't so this issue wasn't resolved by https://github.com/wireapp/wire-ios-images/pull/43 ?

**Comment by billypchan** [comment]
@typfel that PR was solved the tests failures on `wire-ios-images` and this PR fixes the new failure found on `wire-ios`.

File: Source/Model/Message/RasterImages+Protobuf.swift [hunk 1]
```diff
@@ -18,8 +18,21 @@
 
 public extension WireProtos.Asset.Original {
     var hasRasterImage: Bool {
-        guard case .image? = self.metaData,
-            UTType(mimeType: mimeType)?.isSVG == false else { return false }
+        guard case .image? = self.metaData else {
+            return false
+        }
+        
+        #if targetEnvironment(simulator)
```
Comment (typfel) [inline Source/Model/Message/RasterImages+Protobuf.swift:25]: So the root cause issue for this problem is missing arm64 simulator support?

Then let's add a
// FUTUREWORK remove once arm64 simulator support have been added (<link to JIRA ticket>)

**Review by typfel — CHANGES_REQUESTED** [review]

File: Source/Model/Message/RasterImages+Protobuf.swift (hunk 1 above)
Comment (billypchan) [reply Source/Model/Message/RasterImages+Protobuf.swift:25]:
```suggestion
        // FUTUREWORK remove once arm64 simulator support have been added JIRA ticket: SQPIT-583
        #if targetEnvironment(simulator)
```

**Review by typfel — APPROVED** [review]

**Pull request merged**